src/codec/encoder/MayaEncoder.cpp

```diff
@@ -529,6 +515,20 @@ class SerializedGeometry {
 		}     // for all geometries
 	}
 
+	// return the highest required uv set (where a valid texture is present)
+	uint32_t scanValidTextures(const prtx::MaterialPtr& mat) {
+		int8_t highestUVSet = -1;
+		for (const auto& t : TEXTURE_UV_MAPPINGS) {
```

Comment:
The TEXTURE_UV_MAPPINGS and the TextureUvMapping struct are only used here, could be moved inside the class.